decidim-participatory_processes/spec/decidim/commands/destroy_resource_spec.rb

```diff
@@ -0,0 +1,61 @@
+# frozen_string_literal: true
+
+require "spec_helper"
+
+describe Decidim::Commands::DestroyResource do
+  subject { described_class.new(resource, user) }
+
+  let(:organization) { create(:organization) }
+
+  context "when destroying decidim participatory process type" do
```

Comment:
Let's drop decidim from the description, as literally everything in this app is decidim 😄 

```suggestion
  context "when destroying a participatory process type" do
```